File: content-solutions/sorting/adjacent-swaps.java
class Solution {
    public int solution(int[] heights) {
        int[] a = heights.clone();
        int swaps = 0;
        for (int i = 1; i < a.length; i++) {
            for (int j = i; j > 0 && a[j - 1] > a[j]; j--) {
                int t = a[j - 1];
                a[j - 1] = a[j];
                a[j] = t;
                swaps++;
            }
        }
        return swaps;
    }
}

File: content-solutions/sorting/adjacent-swaps.js
function solution(heights) {
  const a = [...heights];
  let swaps = 0;
  for (let i = 1; i < a.length; i++) {
    for (let j = i; j > 0 && a[j - 1] > a[j]; j--) {
      [a[j - 1], a[j]] = [a[j], a[j - 1]];
      swaps++;
    }
  }
  return swaps;
}
